parsers/s01-parse/crowdsecurity/ftp-logs.yaml

```diff
@@ -0,0 +1,21 @@
+onsuccess: next_stage
+name: ftp-logs
+description: "Parse FTP (vsftpd) logs"
+filter: "evt.Parsed.program == 'ftp'"
+#debug: true
+pattern_syntax:
+  FTP_AUTH_FAIL: '%{HTTPDERROR_DATE:timestamp} \[pid %{NUMBER}\] \[%{GREEDYDATA:user}\] FAIL LOGIN: Client "%{IP:source_ip}"'
```

Comment:
if we only support the FTP_FAIL_AUTH message, I guess it's worth saying as well :)